src/force/CabanaPD_Force_PMB.hpp

```diff
@@ -161,6 +161,43 @@ class Force<ExecutionSpace, ForceModel<PMB, Elastic, ModelParams...>>
             double w = 0.25 * model.c * s * s * xi * vol( j );
             W( i ) += w;
             Phi += w * vol( i );
+
+
+            
+            // Calculate forces for virial stress
+            double fx_i = 0.0;
+            double fy_i = 0.0;
+            double fz_i = 0.0;
+
+            double rx, ry, rz;
+            getDistanceComponents( x, u, i, j, xi, r, s, rx, ry, rz );
+
+            // Check if the bond is broken
+            //if ( model.criticalStretch( i, j, r, xi ) && !nofail( i ) && !nofail( j ) )
```

Comment:
This version of the kernel doesn't include fracture, so we can remove this for now. Once it's working we can add below (where you see `Fracture`)

Reply:
removed the bond breakage check from the virial stress logic as we are not recalculating the forces anymore